docs/quickstart/rele_and_emulator.rst

```diff
@@ -0,0 +1,44 @@
+Using Relé and pub/sub emulator
+===========================================================
+
+It can be helpful to be able run the emulator in the our development environment.
+To be able to do that we can follow the steps below:
+
+1) Run Pub/Sub emulator in the cloud-sdk container and map the port 8085.
```

Comment:
Shouldn't this be "Run the Google Cloud Pub/Sub emulator", emphasis on the "the"?